17 10 1

gummek opened the pull request.

MichaelGNairn reviewed (commented):
[Michael Nairn - Code review for 17-10-1.docx](https://github.com/ONSdigital/sdg_data_updates/files/12358307/Michael.Nairn.-.Code.review.for.17-10-1.docx)

@gummek  see review. Great work!

gummek commented:
Thanks for the review @MichaelGNairn I have addressed all of the comments in your review, I tried mutate(Year = as.integer(Year)) in various places in the checks file but it didn't seem to work...

MichaelGNairn reviewed (approved):
Great work Katie! Happy to be merged
